en/rules.md

```diff
@@ -160,4 +160,4 @@ the following will **ALWAYS** be true for scorehunts.
  1. Scorehunt Information will be availible in #score-hunting & the [Public Score Hunt Spreadsheet](https://docs.google.com/spreadsheets/d/1iabjrE_O52rNifvUi8s1tBqzThGqa699pmwZdGZTCN8/edit?usp=sharing)
  2. Scorehunts will **NOT** allow CS changer.
  3. Scorehunts will follow the rules set here.
- 4. Breaking scorehunt rules will incur a normal moderation punishment (See Moderation Policy) + a blacklist from all future scorehunts. 
\ No newline at end of file
+ 4. Breaking scorehunt rules will incur a normal moderation punishment (See Moderation Policy) + a blacklist from all future scorehunts. 
```

Comment:
maybe include that its appealable

Reply:
idk the duration of the ban btw yo can add something like one year if you want for example. that is if you say it is appelable

Reply:
also make it only be one appeal, if you get banned after appleaing one time you can't appeal anymore

ty loki for the idea

Reply:
i'd say 6 months sounds reasonable for something like this, but i'd need your opinions.